src/App.css

```diff
@@ -1,38 +1,45 @@
-.App {
-  text-align: center;
+@font-face {
+  font-family: 'DarkXShadowX';
+  src: url('./css/fonts/DarkXShadowX.ttf')
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+@font-face {
+  font-family: 'El-Milagro-de-Empel';
+  src: url('./css/fonts/El-Milagro-de-Empel.otf')
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+@font-face {
+  font-family: 'oblivion';
+  src: url('./css/fonts/oblivion.ttf')
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+@font-face {
+  font-family: 'PRINP';
+  src: url('./css/fonts/PRINP.ttf')
 }
 
-.App-link {
-  color: #61dafb;
+@font-face {
+  font-family: 'The-Wild-Breath-of-Zelda';
+  src: url('./css/fonts/The-Wild-Breath-of-Zelda.otf')
 }
```

Comment:
NONE of these fonts are set in stone.  I really just downloaded 5 that seemed kinda legible and interesting.  Definitely wouldn't be opposed to trying others!